azure-toolkit-libs/azure-toolkit-auth-lib/src/main/java/com/microsoft/azure/toolkit/lib/auth/model/AccountEntity.java

```diff
@@ -15,15 +16,25 @@
 @Getter
 public class AccountEntity {
 
-    private AuthType type;
+    private AuthMethod method;
+
+    private AzureEnvironment environment;
 
     private String email;
 
     private boolean selected;
 
     private boolean authenticated;
 
-    List<SubscriptionEntity> subscriptions;
+    private List<SubscriptionEntity> subscriptions;
```

Comment:
As we have `List<SubscriptionEntity> subscriptions` and SubscriptionEntity will record whether it is selected, do we need to keep `selectedSubscriptions` and `selectedSubscriptionIds`?

Reply:
selectedSubscriptionIds are only used to pass in selected ids, it should never called after initialized tenants and subscriptions, I will remove the getter for selectedSubscriptionIds